CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
-## [In progresss]
+## [0.1.13] - [In progress]
 
 ### Changed
 - The default values for the tensor network emulator were updated to better ones.
 - the client_id and client_secret was leftover in the Client object even though they are no longer used.
 - Updated the README to also supply the group_id which is mandatory.
+- Updated the default endpoint to `/core-fast` in accordance with infra changes.
```

Comment:
@awennersteen can you specify that all cloud SDK users should switch to using the `core-fast` microservice instead of `core` ?

Reply:
Like now?